mxcubeweb/core/components/samplechanger.py

```diff
@@ -413,181 +408,3 @@ def _gripper_changed(self):
         self.app.server.emit(
             "queue", {"Signal": "update", "message": "all"}, namespace="/hwr"
         )
-
-
-# Disabling C901 function is too complex (19)
-def queue_mount_sample(view, data_model, centring_done_cb, async_result):  # noqa: C901
-    from mxcubeweb.app import MXCUBEApplication as mxcube
-    from mxcubeweb.routes import signals
-
-    HWR.beamline.sample_view.clear_all()
-    logging.getLogger("user_level_log").info("Loading sample ...")
-    log = logging.getLogger("user_level_log")
-
-    loc = data_model.location
-    data_model.holder_length
-
-    robot_action_dict = {
-        "actionType": "LOAD",
-        "containerLocation": loc[1],
-        "dewarLocation": loc[0],
-        "sampleBarcode": data_model.code,
-        "sampleId": data_model.lims_id,
-        "sessionId": HWR.beamline.session.session_id,
-        "startTime": time.strftime("%Y-%m-%d %H:%M:%S"),
-    }
-
-    # devices that can move sample on beam
-    # (sample changer, plate holder)
-    sample_mount_device = HWR.beamline.sample_changer
-
-    mount_from_harvester = mxcube.harvester.mount_from_harvester()
-
-    if (
-        sample_mount_device.get_loaded_sample()
-        and sample_mount_device.get_loaded_sample().get_address() == data_model.loc_str
-    ):
-        return
-
-    if hasattr(sample_mount_device, "__TYPE__"):
-        if sample_mount_device.__TYPE__ in ["Marvin", "CATS"]:
-            element = "%d:%02d" % loc
-            sample = {"location": element, "sampleID": element}
-            mxcube.sample_changer.mount_sample_clean_up(sample)
-        else:
-            sample = {
-                "location": data_model.loc_str,
-                "sampleID": data_model.loc_str,
-            }
-
-            # in this case the sample changer takes the sample from an Harvester
-            # We Harvest the sample and make it ready to load
-            if mount_from_harvester:
-                mxcube.harvester.queue_harvest_sample(data_model)
-
-            try:
-                res = mxcube.sample_changer.mount_sample_clean_up(sample)
-            except RuntimeError:
-                res = False
-            logging.getLogger("user_level_log").info(
-                "Sample loading res: %s" % str(res)
-            )
-
-            # We need to investigate if the comment below is still valid
-            if not res == False:  # noqa: E712
-                # WARNING: explicit test of False return value.
-                # This is to preserve backward compatibility (load_sample was
-                # supposed to return None); if sample could not be loaded, but
-                # no exception is raised, let's skip the sample
-
-                raise queue_entry.QueueSkipEntryException(
-                    "Sample changer could not load sample", ""
-                )
-
-    # Harvest Next sample while loading current
-    if mount_from_harvester and not HWR.beamline.harvester.get_room_temperature_mode():
```

Comment:
Here also,  to be ported here on this PR
https://github.com/mxcube/mxcubecore/pull/971